discovery-handlers/onvif/src/discovery_impl.rs

```diff
@@ -372,7 +372,7 @@ pub mod util {
             time::{Duration, SystemTime},
         };
 
```

Comment:
should try_recv_string use try_recv_from now?

Reply:
Thanks for pointing out that note. I think using the async `recv_from` may be better than `try_recv_from`. It seems preferable to await a response that to poll and loop continuously. If that makes sense to you too, I'll remove the comment